index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="./css/styles.css">
+ <link rel="shortcut icon" href="./assets/image-header/favicon-16x16.png" type="image/x-icon">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+ <title>CSA Home Page</title>
+</head>
+<body>
+   <div class="wrapper">
+    <header class="header-section">
+      <nav class="nav">
+        <span class="open-nav"><i class="material-icons">menu</i></span>
+        <div class="top-nav">
+          <ul class="top-items">
+            <li class="facebook"><a href="#" class="facebook-link"><span class="material-icons">facebook</span></a></li>
+            <li class="twitter"><a href="#" class="twitter-link"><i class="material-icons">whatshot</i></a></li>
+            <li class="language">English</li>
+            <li class="page"><a href="#" class="page-link">My Page</a></li>
+            <li class="sign-out">Logout</li>
+          </ul>
+        </div>
+        <div class="lower-nav">
+          <div class="nav-logo">
+            <a href="./index.html" class="logo-link"><img src="./assets/image-header/capstoneLogo.PNG"
+                alt="project logo" class="logo"></a>
+          </div>
+          <ul class="nav-items">
+            <li class="nav-item"><a href="./about.html" class="nav-link change">about</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">program</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">join</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">sponsor</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">news</a></li>
+          </ul>
+          <span class="campaign"><a href="#" class="campaign-link">CSA campaign</a></span>
```

Comment:
- [ ] From 320px to 450px, there is some white space in the right side which affect all the section in `index.html` file. Please fix that

### Your design

![Screenshot from 2021-07-08 19-48-24](https://user-images.githubusercontent.com/22434660/124950187-d06fba80-e02f-11eb-9399-eb31d3ae80b0.png)

Reply:
Done